src/Console/Commands/MakeUser.php

```diff
@@ -37,13 +36,16 @@ class MakeUser extends Command
      *
      * @return void
      */
-    public function handle()
-    {
+    public function handle() {
         $email = $this->ask("What is the new user's email address?");
         $name = $this->ask("What is the new user's name?") ?: '';
-        $password = bcrypt($this->secret("What is the new user's password? (blank generates a random one)", str_random(32)));
+        $password = $this->secret("What is the new user's password? (blank generates a random one)", str_random(32));
         $sendReset = $this->confirm('Do you want to send a password reset email?');
 
+        if (!$this->option('plaintext')) { // optional
+            bcrypt($password);
```

Comment:
This won't work I don't think; you'll need to assign the `bcrypt` value back to $password. i.e.

```php
if (! $this->option('plaintext')) {
    $password = bcrypt($password);
}
```

I would, however, make this part of the question flow rather than a separate option. `$encrypt = $this->confirm('Should the password be encrypted?');` and default to true